### ISSUE TITLE
Getting official update after rooting?

### ISSUE THREAD

**Issue by eoleumbi** [HappyZ/dpt-tools#83]
First of all, thanks for the great work!

This is just a question (not an issue)
Is there a way to get future update of sony’s official firmware even after rooting?
If possible, brief guide would be helpful.

Below is just my guess!
1. Flash the official PKGs described in wiki PKGs-I-Made,
2. Follow the update instruction from Sony (doing update from Digital Paper App in PC or macOS)

**Comment by HappyZ** [comment]
The mod considers that and you will still be able to do the official
update. But with that, you may loose the root.

So yes, if there's future updates but you want to keep the root, you need
to flash a customized PKG.

**Comment by HappyZ** [comment]
Using the same way you update dpt before.

**Comment by eoleumbi** [comment]
Thanks!

**Comment by junrenshi** [comment]
A new firmware just comes out with a  version number 1.6.  It has some new features such as the TOC support and the zooming by pinching. 

I have a rooted device with modified Applauncher as suggested in the wiki page.  I am wondering whether or not it is safe to update the firmware.  If updated, will it keep the extra functionalities I had added?  In my case, I have installed KOReader and made it launchable from the Applauncher.
